auth/src/test/java/io/grpc/auth/GoogleAuthLibraryCallCredentialsTest.java

```diff
@@ -191,8 +192,9 @@ public void invalidBase64() throws Exception {
   }
 
   @Test
-  public void credentialsFailsWithIoException() throws Exception {
-    Exception exception = new IOException("Broken");
+  public void credentialsFailsWithRetryableRetryableException() throws Exception {
```

Comment:
Any reason to repeat "Retryable" in `credentialsFailsWithRetryableRetryableException` ?